examples/feast/integration_feature_store/feature_repo/feature_transformations.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+import pandas as pd
+
+from feast import FeatureStore, FileSource, PushSource
```

Comment:
Can we switch to just doing:
```python
import feast
```
here please? Then it'll mean that all these classes will be prefixed by `feast.`. I think that will make it clearer to a reader what is feast vs Hamilton.